pkg/frontend/frontend.go

```diff
@@ -32,6 +32,10 @@ import (
 	"github.com/Azure/ARO-RP/pkg/util/recover"
 )
 
+// TODO this const was put in place to disable the ocm routes
```

Comment:
is there a story for this TODO

Reply:
https://msazure.visualstudio.com/AzureRedHatOpenShift/_workitems/edit/15529527/